doc/DeveloperGuide.md

```diff
@@ -94,6 +94,10 @@ Use case ends.
 2. Should be able to hold up to 1000 persons.
 3. Should come with automated unit tests and open source code.
 4. Should favor DOS style commands over Unix-style commands.
+5. Should come with a help feature and various helper messages so that it is usable by a novice who had never had CLI experience before.
```

Comment:
Good NFR because it is precise in defining what is a novice -> someone who never has a CLI experience before.